trackintel/io/file.py

```diff
@@ -424,6 +424,7 @@ def read_trips_csv(*args, columns=None, tz=None, index_col=object(), **kwargs):
         The column names to rename in the format {'old_name':'trackintel_standard_name'}.
         The required columns for this function include: "user_id", "started_at",
         "finished_at", "origin_staypoint_id" and "destination_staypoint_id".
+        An optional column is "geom" (geometry column with start and destination location)
```

Comment:
Here we should describe that the type is "multipoint"

Reply:
I would be good if you could update the example under "examples" in the docstring below

Reply:
Changed this now